app/javascript/gobierto_investments/webapp/components/MapTour.vue

```diff
@@ -142,6 +143,10 @@ export default {
       ] = responses;
 
       this.dictionary = attributesDictionary;
+      this.middleware = new Middleware({
```

Comment:
You're initializating the middleware, but what are you doing then?

Reply:
If I don't initialize the middleware the MapTour fails, because somewhere there is a call to `getPhases` and the `getAttributesByKey` method is not available